### ISSUE TITLE
Since the screen is high resolution, the text is too small to read on MSX0.

### ISSUE THREAD

**Issue by MIN0** [fr3nd/msx-vi#4]
MSX0 is the latest MSX machine that is being made popular through crowdfunding.
However, the included keyboard has a small number of keys, and there is no Ctrl key, so commands such as saving files and ending editing cannot be executed.
The msx-vi you developed is suitable for such MSX0.
We can edit with fewer keys.

However, there is one problem.
The MSX0 is so small that the text is too small to read on high-resolution screens. Can't you create a lower resolution screen version of msx-vi with larger characters?

https://japan.postsen.com/business/46628/The-legendary-8-bit-MSX-personal-computer-is-revived-with-MSX0-Stack-and-it-becomes-a-computer-for-IoT-%E2%80%93-CAMPFIRE-campfire.html

**Comment by MIN0** [comment]
This is what it looks like when you actually operate it (YouTube)

https://youtu.be/r0_QM15MB-c

**Comment by fr3nd** [comment]
Hi @MIN0,

I developed msx-vi 5 years ago, and I haven't touched the code since then. Honestly, I don't thing changing the columns from 80 to 40 would be an easy task, and I no longer have the time to do it.

**Comment by MIN0** [comment]
Hi @fr3nd ,

Thank you very much for your reply despite your busy schedule.
I got it. I think MSX_VI is useful enough in its current state.
New series are emerging in the MSX series, with extremely small screen sizes and even models without screens appearing.
When MSX0_M5stack appeared, it didn't have a Ctrl key, so the appeal was that it could be edited without the Ctrl key of MSX_VI.
When I thought about it, I felt that there was no need to change MSX_VI every time to make such a move.
I'm glad I found MSX_VI.
Thank you very much for your reply.
